demos/embeddings/stellargraph-keras-node2vec.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<blockquote>which actually performs the dot product of the 'input embedding' of the target node and the 'output embedding' of the context node followed by a sigmoid activation</blockquote><p>Doesn't it perform the dot product on the "output embedding" of both the target and context nodes? Also I think using the term "inner product" here might avoid a bit of confusion since we're passing in <code>ip</code>  instead of <code>dot</code>  although they do the same thing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/536/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
I see now that these are referring to the `input_embedding` and `output_embedding` defined in the model, rather than the "input" vs "output" I was thinking of... which still makes me think https://github.com/stellargraph/stellargraph/pull/536#discussion_r404464928 is relevant

Reply:
Thanks for your feedback, Kevin. Here, it is indeed a bit confused to use the item ``input embedding`` and ``output embedding`` without any introduction. I plan to give an graphic illustration to demonstrate the architecture of word2vec, and referring the ``input embedding`` and ``output embedding`` to the input-to-hidden and hidden-to-output weight matrices respectively.